src/server/controllers/AuthController.js

```diff
@@ -136,6 +137,52 @@ class AuthController {
       return next(error);
     };
   }
+
+  /**
+   * Authenticates user token
+   * @method
+   * @memberof AuthController
+   * @static
+   * @return {function} Express middleware function that calls the
+   * next method after authentication has been completed
+   */
```

Comment:
@ayodelevm this JSDoc = _smh_

Also do you think it will make sense changing this to `recoverAccount`